modules/end-user-guide/partials/proc_building-an-image-for-an-ide-based-on-the-intellij-platform.adoc

```diff
@@ -0,0 +1,160 @@
+[id="building-an-image-for-an-ide-based-on-the-intellij-platform_{context}"]
+= Building an image for an IDE based on the IntelliJ Platform
+
+This procedure describes how to build an image for an IDE based on the IntelliJ Platform up to version 2020.3. See the link:https://github.com/che-incubator/jetbrains-editor-images/blob/main/doc/Compatible-IDE.md[list of compatible IDEs].
+
+.Prerequisites
+
+* The build host has at least 2 GB of available RAM.
+
+* The build host has an installation of following tools:
+
+** link:https://www.docker.com/[Docker] version 18.09 or greater, supporting link:https://docs.docker.com/develop/develop-images/build_enhancements/[Docker BuildKit]
+
+** link:https://git-scm.com[Git]
+
+** link:https://www.gnu.org/software/libc/manual/html_node/Getopt.html[GNU getopt]
+
+** link:https://www.gnu.org/software/wget/[GNU wget]
+
+** link:https://openjdk.java.net/projects/jdk/11/[Java Development Kit (JDK) version 11]
+
+** link:https://stedolan.github.io/jq/[jq]
+
+
+.Procedure
+
+. Get a local copy of the link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains Projector Editor Images repository].
++
+----
+$ git clone https://github.com/che-incubator/jetbrains-editor-images
+$ cd jetbrains-editor-images
+----
+
+. Run the build script with parameters:
++
+[subs="+quotes,macros,attributes"]
+----
+$ ./projector build --tag __<tag>__ --url __<URL>__
+----
++
+`--tag __<tag>__`::
+The name and tag to apply to the image after build in `name:tag` format.
++
+`--url __<URL>__`::
+The URL pointing to an archive of the custom IntelliJ-based distribution. The editor version can be at most 2020.3 because Projector doesn't support later versions. The archive must target the Linux platform, be in `+tar.gz+` format, and include JetBrains Runtime (JBR). 
++
+.Building the Docker image with IntelliJ IDEA Community 2020.3.3
+====
+----
+./projector.sh build --tag che-idea:2020.3.3 \
+  --url https://download-cdn.jetbrains.com/idea/ideaIC-2020.3.3.tar.gz
+----
+====
++
+.Building the Docker image with PyCharm Community 2020.3.5
+====
+----
+$ ./projector.sh build --tag che-pycharm:2020.3.5 \
+  --url https://download.jetbrains.com/python/pycharm-community-2020.3.5.tar.gz
+----
+====
++
+.Building the Docker image with WebStorm 2020.3.3
+====
+----
+$ ./projector.sh build --tag che-webstorm:2020.3.3 \
+  --url https://download.jetbrains.com/webstorm/WebStorm-2020.3.3.tar.gz
+----
+====
++
+.Building the Docker image with IntelliJ IDEA Ultimate 2020.2.2
+====
+----
+$ ./projector.sh build --tag che-idea-ultimate:2020.2.2 \
+  --url https://download.jetbrains.com/idea/ideaIU-2020.2.2.tar.gz
+----
+====
+
+. To test the image, run it locally, and navigate to `++http://localhost:8887++` to access the IDE.
```

Comment:
```suggestion
. To test the image, run it locally and navigate to `++http://localhost:8887++` to access the IDE.
```

Reply:
IMB Style
_navigate verb
Use only when a more direct verb, such as "click", "select", "browse", or "go to", is not applicable. Use only to provide direction to a location (for example, "Navigate to the directory that contains the files that you want to edit")._
imho `go to` is easier to understand if you just mean opening a website.

Reply:
```suggestion
. To test the image, run it locally and go to `++http://localhost:8887++` to access the IDE.
```